gc.c

```diff
@@ -814,15 +814,20 @@ typedef struct rb_objspace {
 /* default tiny heap size: 16KB */
 #define HEAP_PAGE_ALIGN_LOG 14
 #define CEILDIV(i, mod) (((i) + (mod) - 1)/(mod))
+#define HEAP_PAGE_ALIGN (1 << HEAP_PAGE_ALIGN_LOG)
+#define HEAP_PAGE_SIZE HEAP_PAGE_ALIGN
```

Comment:
These values had to be moved out of the enum because they're used below in a preprocessor macro.

Reply:
Can we keep the enum the same, but also assign to a preprocessor macro?  I'm not 100% sure why we're using an enum, but it's probably safer not to change it.